tests/integration/test_tlsv1_3/test.py

```diff
@@ -88,10 +90,18 @@ def test_https_wrong_cert():
         execute_query_https("SELECT currentUser()", user="john", cert_name="client2")
     assert "HTTP Error 403" in str(err.value)
 
+    count = 0
     # Wrong certificate: self-signed certificate.
-    with pytest.raises(Exception) as err:
-        execute_query_https("SELECT currentUser()", user="john", cert_name="wrong")
-    assert "unknown ca" in str(err.value)
+    while count <= MAX_RETRY:
+        with pytest.raises(Exception) as err:
+            execute_query_https("SELECT currentUser()", user="john", cert_name="wrong")
```

Comment:
Because this error only an issue for requests that are using an invalid cert, I would like to keep the scope of the retries as strict as possible, especially considering that we are not fixing anything, we just allowing the requests fail in a particular way a few times. E.g.: maybe we can have a race condition in our networking layer that would cause the broken pipe error to show up in other test cases time to time. If we include the retry logic to the `execute_query_https` method, we might not notice those errors.

However if you think that is way to go, I am happy to make the changes.